projects/components/src/utils/test/datagrid/datagrid.wo.ts

```diff
@@ -35,6 +35,15 @@ export class ClrDatagridWidgetObject extends WidgetObject<ClrDatagrid> {
         return this.getNodeText(this.getCell(row, column));
     }
 
+    /**
```

Comment:
Since there's no way for two rows on the same column to return a different value, this should not take the row index as an input. It also doesn't really test is going to be clipped, which is a good thing because this class is not responsible for that. Let's rename it to 

```typescript
columnHasClippedTextDirective(column: number): boolean
```

We should try querying for the directive instead of using what seems to be an internal feature, it's an array of `any` without much documentation.